docs/_documentations/metrics-dashboard.md

```diff
@@ -1,31 +1,31 @@
 ---
 layout: docs
-title: Understanding Application Metrics
-description: Understanding Application Metrics
+title: Understanding the Metrics Dashboard
+description: Understanding the Metrics Dashboard
 keywords: import, help, metrics, Swift, Node.js, Java, performance monitoring, app monitor, dashboard, CPU, HTTP incoming requests, memory, HTTP throughput, Heap, HTTP outbound requests, loop times, other requests, run load, HTTP Requests, environment, resource usage, profiling, enabling, graph
 duration: 1 minute
-permalink: appmetrics
+permalink: metrics-dashboard
 type: document
 parent:
 order:
 ---
 
-# Understanding Application Metrics
+# Understanding the Metrics Dashboard
 
-## What is Application Metrics?
+## What is the Metrics Dashboard?
 
-Application Metrics instruments Java, Node.js, and Swift runtimes for performance monitoring and provides the monitoring data through an API. You can also visualize data with the Application Metrics Dashboard. For more information, see [Application Metrics on GitHub](https://github.com/RuntimeTools/appmetrics).
+The Metrics Dashboard instruments Java, Node.js, and Swift runtimes for performance monitoring and provides the monitoring data through an API. You can also visualize data with the Metrics Dashboard. For more information, see [Application Metrics on GitHub](https://github.com/RuntimeTools/appmetrics).
 
-## Application Metrics types
+## Metrics Dashboard types
 
-Application Metrics is available in Java, Node.js, and Swift. You can monitor the projects that you create in these languages.
+The Metrics Dashboard is available in Java, Node.js, and Swift. You can monitor the projects that you create in these languages.
```

Comment:
Should the be `is available for` (again I think this is because the focus has changed from application metrics to the metrics dashboard).

Reply:
Yes.